src/ripple/crypto/impl/KeyType.cpp

```diff
@@ -1,39 +0,0 @@
-//------------------------------------------------------------------------------
```

Comment:
This file needs to be removed from CMakeLists.txt or the non unity build will fail.

Reply:
Gah, I thought I did this.